qa/rolling-upgrade/build.gradle

```diff
@@ -203,6 +205,13 @@ task testAgainstTwoThirdsUpgradedCluster(type: StandaloneRestIntegTestTask) {
         }
     }
 
+    // Excluding batch ingestion tests because we introduce this feature in 2.14
+    if (ext.neural_search_bwc_version.startsWith("2.9") || ext.neural_search_bwc_version.startsWith("2.10") || ext.neural_search_bwc_version.startsWith("2.11") || ext.neural_search_bwc_version.startsWith("2.12") || ext.neural_search_bwc_version.startsWith("2.13")){
```

Comment:
Do you know why `k-NN radial search` is not excluded here?

Reply:
That's a good question, @junqiu-lei do you happen to know why  `k-NN radial search` is not excluded under some rolling upgrade tasks? Is it intentional?

Reply:
If k-NN is not excluded, then there is no need to exclude batch ingestion test.

Reply:
They are independent, we'd better add based on our own need.

Reply:
Radial search is only available since OpenSearch version 2.14.